reference/supervisor.md

```diff
@@ -2020,35 +1922,19 @@ wb_supervisor_world_reload()
 
 | name | service/topic | data type | data type definition |
 | --- | --- | --- | --- |
-| `/supervisor/world_load` | `service` | [`webots_ros::set_string`](ros-api.md#common-services) | |
-| `/supervisor/world_save` | `service` | [`webots_ros::set_string`](ros-api.md#common-services) | |
-| `/supervisor/world_reload` | `service` | [`webots_ros::get_bool`](ros-api.md#common-services) | |
+| `/supervisor/simulation_reset` | `service` | `webots_ros::get_bool ` | |
 
 %tab-end
 
 %end
 
 ##### Description
 
-*Load, save or reload the current world.*
-
-The `wb_supervisor_world_load` function sends a request to the simulator process, asking it to stop the current simulation and load the world given in argument immediately.
-As a result of changing the current world, the supervisor process and all the robot processes are terminated and the new one are restarted with the new world.
-You may wish to save some data in a file from your supervisor program in order to reload it from the new world.
-
-The `wb_supervisor_world_save` function saves the current world.
-The `filename` parameter defines the path to the target world file.
-It should end with the `.wbt` extension.
-It can be defined either as an absolute path, or as a path relative to the current supervisor controller.
-If NULL, the current world path is used instead (e.g., a simple save operation).
-The boolean return value indicates the success of the save operation.
-Be aware that this function can overwrite silently existing files, so that the corresponding data may be lost.
-
-> **Note** [C++, Java, Python, MATLAB]: In the other APIs, the `Robot.worldSave` function can be called without argument.
-In this case, a simple save operation is performed.
+*reset the simulation*
 
-The `wb_supervisor_world_reload` function sends a request to the simulator process, asking it to reload the current world immediately.
-As a result of reloading the current world, the supervisor process and all the robot processes are terminated and restarted.
+The `wb_supervisor_simulation_reset` function sends a request to the simulator process, asking it to reset the simulation immediately.
+The reset process is explained in detail in the [User Guide](https://www.cyberbotics.com/doc/guide/the-user-interface#file-menu).
+As a result of the reset, the supervisor process and all the robot processes are terminated and restarted.
```

Comment:
This could be improved as there may be several supervisor processes &rarr; "all the supervisor and robot controller processes are terminated and restarted."